src/chrome/content/rules/hs-mannheim.de.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	Invalid Certificate:
+		mim.hs-mannheim.de
+		www.mechatronik.hs-mannheim.de
+		www.analogtechnik.hs-mannheim.de
+		www.emb.hs-mannheim.de
+		www.fsw.hs-mannheim.de
+		www.fotovoltaik.hs-mannheim.de
+	
+	Time Out:
+		www.gestaltung.hs-mannheim.de
+-->		
+<ruleset name="Hochschule Mannheim (partial)">
+
+	<!-- hs-mannheim.de -->
+	<target host="www.hs-mannheim.de"/>
```

Comment:
`www` is already covered by the wildcard, but the wildcard would also break all non-working hosts listed above.